components/Header/NavItem.js

```diff
@@ -41,17 +43,41 @@ const NavItem = ({
     if (image) {
       return <NavImage src={image} imageWidth={imageWidth} style={style} />;
     }
-
     return title;
   };
 
   return (
-    <Link href={href} passHref>
-      <StyledLink onClick={onClick} color={color} target={target}>
-        {displayedLink()}
-      </StyledLink>
-    </Link>
+    <>
+      {isLocal ? (
+        <Link href={href} passHref>
```

Comment:
@saragibby why does this need be a next link?

Reply:
Using `Link` keeps it from a full refresh and will just fill the content within `InnerPage`, so it's a nicer experience.